third_party/accessibility/ax/platform/ax_platform_node_win.cc

```diff
@@ -13,46 +13,36 @@
 #include <string>
 #include <unordered_set>
 #include <utility>
-#include <vector>
-
-#include "base/json/json_writer.h"
-#include "base/lazy_instance.h"
-#include "base/metrics/histogram_functions.h"
-#include "base/numerics/safe_conversions.h"
-#include "base/strings/string_number_conversions.h"
-#include "base/strings/string_util.h"
-#include "base/strings/utf_string_conversions.h"
-#include "base/threading/thread_task_runner_handle.h"
-#include "base/values.h"
+
+#include "ax/ax_action_data.h"
+#include "ax/ax_active_popup.h"
+#include "ax/ax_enum_util.h"
+#include "ax/ax_mode_observer.h"
+#include "ax/ax_node_data.h"
+#include "ax/ax_node_position.h"
+#include "ax/ax_role_properties.h"
+#include "ax/ax_tree_data.h"
 #include "base/win/enum_variant.h"
 #include "base/win/scoped_bstr.h"
 #include "base/win/scoped_safearray.h"
 #include "base/win/scoped_variant.h"
 #include "base/win/variant_vector.h"
-#include "skia/ext/skia_utils_win.h"
-#include "third_party/iaccessible2/ia2_api_all.h"
-#include "third_party/skia/include/core/SkColor.h"
-#include "ui/accessibility/accessibility_features.h"
-#include "ui/accessibility/accessibility_switches.h"
-#include "ui/accessibility/ax_action_data.h"
-#include "ui/accessibility/ax_active_popup.h"
-#include "ui/accessibility/ax_constants.mojom.h"
-#include "ui/accessibility/ax_enum_util.h"
-#include "ui/accessibility/ax_mode_observer.h"
-#include "ui/accessibility/ax_node_data.h"
-#include "ui/accessibility/ax_node_position.h"
-#include "ui/accessibility/ax_role_properties.h"
-#include "ui/accessibility/ax_tree_data.h"
-#include "ui/accessibility/platform/ax_fragment_root_win.h"
-#include "ui/accessibility/platform/ax_platform_node_delegate.h"
-#include "ui/accessibility/platform/ax_platform_node_delegate_utils_win.h"
-#include "ui/accessibility/platform/ax_platform_node_textchildprovider_win.h"
-#include "ui/accessibility/platform/ax_platform_node_textprovider_win.h"
-#include "ui/accessibility/platform/ax_platform_relation_win.h"
-#include "ui/accessibility/platform/uia_registrar_win.h"
-#include "ui/base/win/atl_module.h"
-#include "ui/display/win/screen_win.h"
-#include "ui/gfx/geometry/rect_conversions.h"
+
+#include "ax_fragment_root_win.h"
+#include "ax_platform_node_delegate.h"
+#include "ax_platform_node_delegate_utils_win.h"
+#include "uia_registrar_win.h"
+
+#include "base/logging.h"
+#include "base/win/atl_module.h"
+#include "gfx/geometry/rect_conversions.h"
+
+// From ax.constants.mojom
+namespace ax {
+namespace mojom {
+const int32_t kUnknownAriaColumnOrRowCount = -1;
```

Comment:
Is this used anywhere?

Reply:
Yes, in `get_RowCount` and `get_ColumnCount`